omero/users/whatsnew.txt

```diff
@@ -1,12 +1,13 @@
 What's new for OMERO 5.1
 ========================
 
-From a user's perspective, OMERO 5.1 introduces units and some other
-stuff that others **will** write about.
-
-- Moving and deleting data is now faster.
-
-- There is **going to be** a new :help:`Getting Started guide
-  <getting-started-5.html>` for OMERO 5.1.
-
-.. note:: :doc:`demo-server` **is now available for OMERO 5.1.**
+From a user's perspective, OMERO 5.1 introduces support for units and new more
+flexible user-added metadata. Other updates and new features include:
```

Comment:
Enclose "more flexible" in commas? Just a thought -- I am not sure if even I think it would help -- leave as-is if you prefer.